$NetBSD: patch-eg,v 1.1 2010/07/11 23:51:16 dholland Exp $

Fix build with recent xextproto, where the shape extension prototypes
appear to have been moved to their own header file.

--- server/shape.c~	1997-09-27 21:10:13.000000000 +0000
+++ server/shape.c
@@ -33,6 +33,7 @@
 #include <X11/Xutil.h>
 #include <X11/Xproto.h>
 #include <X11/extensions/shape.h>
+#include <X11/extensions/shapeproto.h>
 #include "xmx.h"
 #include "ptc.h"
 #include "incl/shape.pvt.h"
